src/Layers/xrRenderDX10/dx10HW.h

```diff
@@ -0,0 +1,86 @@
+#pragma once
+
+#include "Layers/xrRender/HWCaps.h"
+#include "xrCore/ModuleLookup.hpp"
+#include "SDL.h"
+#include "SDL_syswm.h"
+
+#if !defined(_MAYA_EXPORT)
+#include "Layers/xrRender/stats_manager.h"
+#endif
+
+class CHW
+    : public pureAppActivate,
+      public pureAppDeactivate
+{
+public:
+    CHW();
+    ~CHW();
+
+
```

Comment:
Please, remove this one empty line. Two empty lines is excessive, one is enough.